README.md

```diff
@@ -785,6 +824,10 @@ organizations and sites within them, representing the full hierarchy of
 awardees > organizations > sites. No pagination, syncing, or filtering is currently
 supported on this endpoint.
 
+Excepts an `_inactive=true` parameter to allow the addition of inactive sites to
```

Comment:
Excepts -> Accepts.

You may want to say first that by default, only sites with a status of ACTIVE are included in the response.

Reply:
😆 done.